### PR TITLE
I added me - Tim Smith

### PR DESCRIPTION
**Pull request by timscodebase**

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -11440,6 +11440,44 @@ module.exports = [
       'Blogger',
     ],
   },
+  {
+    name: 'Tim Smith',
+    description:
+      'I am simply a web developer. I love web development.  I am not currently making money with it, but I write code every day with glee',
+    url: 'https://timsmith.tech/uses',
```
Comment (HugoDF) [inline src/data.js:11447]: This 404s, in order for you to be added to awesome-uses, you'll need a working (& filled out) uses page

**Review by HugoDF — CHANGES_REQUESTED** [review]

**Comment by timscodebase** [comment]
I added my /uses page.  I did not before because I am in the process of creating a new site new-tim.netlify.app

**Review by HugoDF — APPROVED** [review]

File: src/data.js [hunk 2]
```diff
@@ -12089,6 +12089,43 @@ module.exports = [
...
   {
+    name: 'Tim Smith',
+    description:
+      'I am simply a web developer. I love web development.  I am not currently making money with it, but I write code every day with glee',
+    url: 'https://timsmith.tech/uses',
+    twitter: 'timsmith23',
+    emoji: '🌈',
+    country: '🇺🇸',
+    computer: 'linux',
+    phone: 'iPhone',
```
Comment (gatarelib) [inline src/data.js:12100]: `Error: "phone" must be one of [iphone, android, windowsphone, flipphone]`

**Review by gatarelib — CHANGES_REQUESTED** [review]

**Comment by BlakeCampbells** [comment]
Closing due to inactivity.

**Pull request closed without merging**